WeakAuras/WeakAuras.lua

```diff
@@ -1356,6 +1356,14 @@ local function GetInstanceTypeAndSize()
     local difficultyInfo = Private.difficulty_info[difficultyIndex]
     if difficultyInfo then
       size, difficulty = difficultyInfo.size, difficultyInfo.difficulty
+    elseif size == "arena" then
+      if C_PvP.IsRatedArena() and not IsArenaSkirmish() then
+        size = "ratedarena"
+      end
+    elseif size == "pvp" then
+      if C_PvP.IsRatedBattleground() then
```

Comment:
IsRatedArena & IsRatedBattleground are not in C_PVP table on classic

Reply:
Oh. There is no rated arena in classic, right?

Reply:
There is no arena at all (there will be in TBC), but shouldn't it be checked in case someone import an aura from retail ?